src/labkey/Ajax.ts

```diff
@@ -252,11 +252,18 @@ function downloadFile(xhr: XMLHttpRequest, config: any): void {
         // parse the filename out of the Content-Disposition header. Example:
         //   Content-Disposition: attachment; filename=data.xlsx
         // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition
+        // also
+        //   Content-Disposition: attachment; filename*=UTF-8''filename.xlsx
         const disposition = xhr.getResponseHeader('Content-Disposition');
         if (disposition && disposition.indexOf('attachment') !== -1) {
-            const matches = /filename[^;=\n]*=((['"]).*?\2|[^;\n]*)/.exec(disposition);
-            if (matches != null && matches[1]) {
-                filename = matches[1].replace(/['"]/g, '');
+            const matches = /filename\*?=([^']*'')?(['"].*?['"]|[^;\n]*)/.exec(disposition);
+            if (matches) {
+                if (matches.length > 2) {
```

Comment:
filename* is always second?

Reply:
(filename\*?) could be its own group

Reply:
It's legal to have both attributes for backward compatibility e.g. filename=file.jpg; filename*=UTF8''%%UNICODE.jpg

Of course, we're not doing that right now.